crates/nu-cmd-extra/src/extra/bits/shift_right.rs

```diff
@@ -56,29 +77,24 @@ impl Command for BitsShr {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let bits: usize = call.req(engine_state, stack, 0)?;
+        let bits: i64 = call.req(engine_state, stack, 0)?;
         let signed = call.has_flag(engine_state, stack, "signed")?;
-        let number_bytes: Option<Spanned<String>> =
+        let number_bytes: Option<Spanned<i64>> =
```

Comment:
Not really sure what if you use `usize`?  Because it only support 1, 2, 4, 8, no negative number

Reply:
agree with this as well